tools/ci_credentials/ci_credentials/main.py

```diff
@@ -1,43 +1,62 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
 import json
-import os
 import sys
 from json.decoder import JSONDecodeError
 
+import click
 from ci_common_utils import Logger
-from . import SecretsLoader
+
+from . import SecretsManager
 
 logger = Logger()
 
 ENV_GCP_GSM_CREDENTIALS = "GCP_GSM_CREDENTIALS"
 
 
 # credentials of GSM and GitHub secrets should be shared via shell environment
-
-def main() -> int:
-    if len(sys.argv) != 2:
-        return logger.error("uses one script argument only: <unique connector name>")
-
+@click.group()
+@click.argument("connector_name")
+@click.option("--gcp-gsm-credentials", envvar="GCP_GSM_CREDENTIALS")
```

Comment:
Confirming that GCP_GSM_CREDENTIALS has read and write access?

Reply:
I'm not allowed to check in GitHub which service account is used in this env var in our actions. But according to the service account I see in our IAM console, I'm pretty sure the currently used SA has sufficient permissions to perform all the operations that this CLI can do: read secret values, add a secret version, disable a secret version.